anyway/database.py

```diff
@@ -1,3 +1,6 @@
-from anyway.app_and_db import db
-
-Base = db.Model
+try:
+    from anyway.app_and_db import db
+    Base = db.Model
+except ModuleNotFoundError:
+    from sqlalchemy.ext.declarative.api import declarative_base
+    Base = declarative_base()
```

Comment:
As far as I can see, this module is only used from `models.py`. What is the need for a dedicated module?

Reply:
to be handled in #1876